landing-pages/site/content/en/ecosystem/_index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Ecosystem"
+linkTitle: "Ecosystem"
+menu:
+  main:
+    weight: 30
+---
+
+&nbsp;
+&nbsp;
+
+# Ecosystem
+
+Feel free to contribute any resources or tools that you find useful when learning or using Airflow.
```

Comment:
I think we should also make a disclaimer here  - something along the lines (I looked at other similar pages for an inspiration):

`
Those resources and services are not maintained, nor endorsed by the Apache Airflow Community and Apache Airflow project. Use them at your sole discretion. The community does not verify the licences nor validity of those tools, so it's your responsibility to verify them.

If you would you like to be included on this page, please reach out to the [Apache Airflow dev or user mailing list](https://airflow.apache.org/community/) and let us know or simply open Pull Request to that page.
`

@kaxil @leahecole @aijamalnk @ryw  - can you please help with correct statement/wording here? I think It's very important to be clear about the wording here and make it super-clear those are not the community managed resources.

Reply:
LGTM, maybe add maintained by the PMC (as shown below) ?

Those resources and services are not maintained, nor endorsed by the Apache Airflow Community and Apache Airflow project (maintained by the Airflow PMC). Use them at your sole discretion. The community does not verify the licences nor validity of those tools, so it's your responsibility to verify them.

Reply:
But is not the project maintained by the Commiters @kaxil  ?

Reply:
Yeah maybe we could list both committers and PMC. I was just following https://www.apache.org/foundation/how-it-works.html#pmc which says "The role of the PMC from a Foundation perspective is oversight. The main role of the PMC is not code and not coding - but to ensure that all legal issues are addressed, that procedure is followed" but yeah let's include both to avoid nay confusion.

Reply:
So then I'll add...

"Those resources and services are not maintained, nor endorsed by the Apache Airflow Community and Apache Airflow project (maintained by the Committers and the Airflow PMC). Use them at your sole discretion. The community does not verify the licences nor validity of those tools, so it's your responsibility to verify them.

If you would you like to be included on this page, please reach out to the [Apache Airflow dev or user mailing list](https://airflow.apache.org/community/) and let us know or simply open Pull Request to that page."

Is that okey?

Reply:
Perfect

Reply:
thanks Jarek, and these disclaimers are necessary. I think what you provided is the standard wording, and I like it best.

Those resources and services are not maintained, nor endorsed by the Apache Airflow Community and Apache Airflow project. Use them at your sole discretion. The community does not verify the licences nor validity of those tools, so it's your responsibility to verify them.

If you would you like to be included on this page, please reach out to the Apache Airflow dev or user mailing list and let us know or simply open Pull Request to that page.